lib/bulbs-elements/settings.js

```diff
@@ -0,0 +1,31 @@
+let settings = {
+  base: {
+    BULBS_ELEMENTS_ONIONSTUDIOS_GA_ID: 'UA-223393-14',
+    BULBS_ELEMENTS_COMSCORE_ID: '6036328',
+    RESP: 'vmap1',
+  },
+  development: {
+    FREEWHEEL_NETWORK_ID: '111976',
```

Comment:
I think these network IDs should actually be stored in each property's python settings and then passed into the the bulbs-element somehow, maybe even as a namespaced variable on `window`.  I don't believe the `NODE_ENV` gives you what you need to integrate into each of the bulbs properties, but I'd want to @collin to weigh in.